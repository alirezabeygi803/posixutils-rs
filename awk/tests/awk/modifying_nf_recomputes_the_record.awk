{
	print NF;
	print $0;
	NF = 10;
	print $0;
	NF = 3;
	print $0;
	NF = 1;
	print $0;
	NF = 5;
	print $0;
}