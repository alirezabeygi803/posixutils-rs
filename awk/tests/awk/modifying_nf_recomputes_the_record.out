4
1 Jane janitor 30
1 Jane janitor 30      
1 Jane janitor
1
1    
4
2 Smith sailor 45
2 Smith sailor 45      
2 Smith sailor
2
2    
4
3 Roselyn radiologist 28
3 Roselyn radiologist 28      
3 Roselyn radiologist
3
3    
4
4 Megan machinist 32
4 Megan machinist 32      
4 Megan machinist
4
4    
4
5 Celine clergy 87
5 Celine clergy 87      
5 Celine clergy
5
5    
4
6 Fredrick farmer 19
6 Fredrick farmer 19      
6 Fredrick farmer
6
6    
4
7 Benjamin programmer 53
7 Benjamin programmer 53      
7 Benjamin programmer
7
7    
